apps/workflow-server/gcc-workflow-server-facade/gcc-restclient-facade-default/src/main/java/com/coremedia/labs/translation/gcc/facade/def/DefaultGCExchangeFacade.java

```diff
@@ -167,7 +170,7 @@ public String uploadContent(String fileName, Resource resource) {
   }
 
   @Override
-  public long submitSubmission(@Nullable String subject, ZonedDateTime dueDate, Locale sourceLocale, Map<String, List<Locale>> contentMap) {
+  public long submitSubmission(@Nullable String subject, String comment, ZonedDateTime dueDate, String workflow, String submitter, Locale sourceLocale, Map<String, List<Locale>> contentMap) {
```

Comment:
(at)Nullable annotations are missing for new parameters. They should also be added at the implemented interface.